contrib/platform/test/com/sun/jna/platform/win32/GDI32UtilTest.java

```diff
@@ -0,0 +1,52 @@
+/* Copyright (c) 2015 Michael Freeman, All Rights Reserved
+ * 
+ * This library is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU Lesser General Public
+ * License as published by the Free Software Foundation; either
+ * version 2.1 of the License, or (at your option) any later version.
+ * 
+ * This library is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+ * Lesser General Public License for more details.  
+ */
+package com.sun.jna.platform.win32;
+
+import java.awt.image.BufferedImage;
+import java.util.ArrayList;
+import java.util.List;
+
+import com.sun.jna.platform.win32.WinDef.HWND;
+
+import junit.framework.TestCase;
+
+public class GDI32UtilTest extends TestCase {
+
+	public static void main(String[] args) {
```

Comment:
In general I recommend using Junit 4.x annotations - e.g., _@Test_ for newly written tests. To this end, consider extending _AbstractWin32TestSupport_ and attaching _@Test_ annotations.